complement.ts

```diff
@@ -12,10 +12,10 @@ import { not } from './not.ts'
 // @types
 type Complement = (<T extends any[]>(
   a: (...args: T) => boolean,
-) => (...args: T) => boolean) &
-  ((fn?: PH) => Complement)
+) => (...args: T) => boolean)
```

Comment:
this must be in 1 line only as this type contains only one case
Also, remove extra brackets if there is any